packages/calling-component-bindings/package.json

```diff
@@ -33,6 +33,7 @@
     "@internal/acs-ui-common": "2.0.0-beta.0",
     "@internal/calling-stateful-client": "2.0.0-beta.0",
     "@internal/react-components": "2.0.0-beta.0",
+    "@internal/react-components-v2": "2.0.0-beta.0",
```

Comment:
Should it be "2.0.0-beta.1"?

Reply:
Good question! The reason it's `.0` is the at release time we bump the version, so at release time these will be bumped to `.1` and the first release will be 2.0.0-beta.1